src/pytorch_lightning/CHANGELOG.md

```diff
@@ -221,6 +221,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - Removed the deprecated `Trainer.use_amp` and `LightningModule.use_amp` attributes ([#14832](https://github.com/Lightning-AI/lightning/pull/14832))
 
+- Removed the deprecated `Trainer.run_stage` in favor of `Trainer.{fit,validate,test,predict}`
```

Comment:
maybe I missed it but why no one is adding the PR link for these PRs?

Reply:
Thanks for catching this! I believe we forgot this. Should I open a new PR to add this?

Reply:
I added this in another PR :)